iotdb-core/confignode/src/test/java/org/apache/iotdb/confignode/manager/load/cache/detector/PhiAccrualDetectorTest.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iotdb.confignode.manager.load.cache.detector;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class PhiAccrualDetectorTest {
```

Comment:
1. rename to DetecterTest
2. to compare the difference between PhiAccrual and Fixed under same heartbeat interval list，to list some case dentifing each other's strengths and weaknesses, for example, Phi may detect the failure earlier, but there will be false positives
3. Each test should be complemented with a description of what the HeartbeatInterval corresponds to in the real world (e.g., a node going down, or a 5s/10s gc, etc.).

Reply:
Done